### PR TITLE
fix(react/configure): unsetting of pending adapter args

### PR DESCRIPTION
**Pull request by tdeekens**

### CODE REVIEW

File: packages/react/modules/components/configure-adapter/configure-adapter.js [hunk 1]
```diff
@@ -69,10 +69,20 @@ export default class ConfigureAdapter extends PureComponent<Props, State> {
...
+       *   a batched `setState` otherwise outdated `adapterArgs` may be
+       *   applied as the `setState` is actually performed later.
+       */
+      this.pendingAdapterArgs = null;
+
+      return {
+        ...prevState,
+        appliedAdapterArgs: nextAdapterArgs,
+      };
+    });
```
Comment (dferber90) [inline packages/react/modules/components/configure-adapter/configure-adapter.js]: I don't fully get the comment... but I'm missing a ton of context anyways.

Would one of these two solutions be preferable?

```js
this.pendingAdapterArgs = null;
this.setState(prevState => {})
```

```js
this.setState(
  prevState => {},
  () => { this.pendingAdapterArgs = null; }
)
```

Just a suggestion as I've never seen a side-effect within the `setState`.

Comment (tdeekens) [reply packages/react/modules/components/configure-adapter/configure-adapter.js]: Oh shit. Thanks. Will do

File: packages/react/modules/components/configure-adapter/configure-adapter.spec.js [hunk 2]
```diff
@@ -444,7 +464,7 @@ describe('interacting', () => {
       beforeEach(() => {
         wrapper.instance().setPendingAdapterArgs({
           adapterArgs: nextAdapterArgs,
-          options: { exashouldOverwritect: false },
```
Comment (emmenko) [inline packages/react/modules/components/configure-adapter/configure-adapter.spec.js]: LoL 😅

Comment (tdeekens) [reply packages/react/modules/components/configure-adapter/configure-adapter.spec.js]: Yeah late night coding doesn't do anyone good :)

**Pull request merged**